src/olympia/devhub/utils.py

```diff
@@ -292,13 +292,15 @@ def create_file_upload_tasks(self, upload_pk, is_mozilla_signed):
 
 def extract_theme_properties(addon, channel):
     version = addon.find_latest_version(channel)
+    print('version', version)
     if not version:
         return {}
     try:
         parsed_data = parse_xpi(
             version.file.file.path, addon=addon, user=core.get_user()
         )
-    except (ValidationError, ValueError):
+    except (ValidationError, ValueError) as exc:
+        print(f'Error parsing xpi: {exc}')
```

Comment:
ditto

Reply:
I actually think we should leave that. When I hit that error, it was totally silent and gave no indication of what was wrong.

Reply:
Use a logger then.

Reply:
Probably something like (re-using the `log` logger defined at the top of this file)

```suggestion
        log.exception('Error parsing xpi', exc_info=exc)
```

Reply:
Done.